pkgs/applications/networking/discordo/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub, nix-update-script, makeWrapper, xsel
+, wl-clipboard }:
+
+buildGoModule rec {
+  pname = "discordo";
+  version = "unstable-2023-04-07";
+
+  src = fetchFromGitHub {
+    owner = "ayn2op";
+    repo = pname;
+    rev = "f8c58057945b1ded2f78dc0956ea25aa281a0b31";
+    hash = "sha256-FUSPQK4rB0J89s+I7yhF8SQ/Q9uygQSCF9o6ltYxOk4=";
+  };
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version=branch" ];
+  };
+
+  vendorHash = "sha256-fLhyyIChqh+eEzht3CSLPfx6glw0YhiTb9PsbWJafWQ=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-s" "-w" ];
+
+  # Clipboard support on X11 and Wayland
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/discordo \
+      --prefix PATH : ${lib.makeBinPath [ xsel wl-clipboard ]}
+  '';
+
+  meta = with lib; {
+    description =
+      "A lightweight, secure, and feature-rich Discord terminal client";
```

Comment:
```suggestion
    description = "Lightweight, secure, and feature-rich Discord terminal client";
```

Reply:
What formatter would you suggest? Emacs (using `nixfmt` I believe) formats long lines like that

Reply:
nixpkgs-fmt or change the line length of nixfmt to 120 or something similar to not generate such silly short lines.